### PR TITLE
Feature/delivery page mobile

### PR DESCRIPTION
**Pull request by mrSelieznov**
#135 Task
delivery page with mobile wive cards

### CODE REVIEW

File: src/components/RenderResults.tsx [hunk 1]
```diff
@@ -7,6 +7,9 @@ import '../styles/RenderResults.scss';
 import CardInfo from './CardInfo';
 import { MovieCard } from '../models/index';
 
+import useDeviceDetect from 'src/hooks/useDeviceDetect';
```
Comment (oleksandrstarov) [inline src/components/RenderResults.tsx:10]: use relative import

File: src/components/RenderResults.tsx [hunk 2]
```diff
@@ -15,7 +18,7 @@ const RenderResults = ({ list }: RenderResultsArrList) => {
   const [tailState, setTailState] = useState(true);
 
   const handlerTail = () => setTailState(!tailState);
-
+  const mobileView = useDeviceDetect()
```
Comment (oleksandrstarov) [inline src/components/RenderResults.tsx:21]: it is boolean - use is prefix isMobileView

File: src/components/RenderResults.tsx [hunk 3]
```diff
@@ -37,7 +40,20 @@ const RenderResults = ({ list }: RenderResultsArrList) => {
...
+
+            if (tailState) {
+              return <CardInfo tailWide={false} number={item.id} />;
+            }
+
+            return <div className="small-card-wrapper"><SmallInfoCard id={item.id} /></div>
+
+          }
+
+
```
Comment (oleksandrstarov) [inline src/components/RenderResults.tsx:56]: check formatting - too many empty space

**Review by oleksandrstarov — APPROVED** [review]
Fix comments and merge

**Review by solomiia-sahala — APPROVED** [review]

**Pull request merged**